db/migrations/postgres/000001_create_teams.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS teams (
```

Comment:
There is a groupconstrained column which seems to be missing: 

`sqlStore.CreateColumnIfNotExistsNoDefault("Teams", "GroupConstrained", "tinyint(4)", "boolean")`